src/test/java/guitests/UseGlobalConfigsTest.java

```diff
@@ -96,10 +95,9 @@ public void globalConfigTest() {
         Label renameButton1 = filterPanel1.getRenameButton();
         click(renameButton1);
         type("Dummy 2 panel");
-        PanelNameTextField renameTextField1 = filterPanel1.getRenameTextField();
-        waitUntilNodeAppears(renameTextField1);
-        assertEquals("Dummy 2 panel", renameTextField1.getText());
-        pushKeys(KeyCode.ENTER);
+        push(KeyCode.ENTER);
+        assertEquals("Dummy 2 panel", filterPanel1.getNameText().getText());
+        PlatformEx.waitOnFxThread();
```

Comment:
Same here. We've already asserted the condition, so there might not be a need to wait.